actors/verifreg/src/lib.rs

```diff
@@ -378,77 +378,84 @@ impl Actor {
         let mut datacap_claimed = DataCap::zero();
         let mut ret_gen = BatchReturnGen::new(params.sectors.len());
         let all_or_nothing = params.all_or_nothing;
-        rt.transaction(|st: &mut State, rt| {
-            let mut claims = st.load_claims(rt.store())?;
-            let mut allocs = st.load_allocs(rt.store())?;
+        let sector_claims = rt
```

Comment:
Nit: since we're already reaching outside the local scope anyway to mutate `ret_gen`, I thnk simpler to just follow the same pattern and init sector_allocations outside the closure.